docs/jwst/tso_photometry/description.rst

```diff
@@ -31,6 +31,14 @@ the times will be computed from the exposure start time, the integration time,
 and the number of integrations.  In either case, the times are
 Modified Julian Date, time scale UTC.
 
+If NaNs exist in the source or background annulus the NaNs are masked out
+and the TSO mode photometry will not will not return NaN.
```

Comment:
This looks good but I think you should put some sort of explicit comment or warning that the resulting fluxes will be underestimated in that case